pkg/cmd/run.go

```diff
@@ -674,7 +674,9 @@ func init() {
 	forwardSignals := !isatty.IsTerminal(os.Stdout.Fd())
 
 	runCmd.Flags().StringP("project", "p", "", "project (e.g. backend)")
+	runCmd.RegisterFlagCompletionFunc("project", projectIDsValidArgs)
 	runCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
+	runCmd.RegisterFlagCompletionFunc("config", configNamesValidArgs)
```

Comment:
I think autocomplete would be really nice for the `name-transformer` and `mount-template` flags on this command.

Reply:
Yes! That looks great.